scio-examples/src/main/java/org/apache/beam/examples/complete/TfIdf.java

```diff
@@ -364,15 +374,17 @@ public void processElement(ProcessContext c) {
               ParDo.of(
                   new DoFn<KV<String, CoGbkResult>, KV<String, KV<URI, Double>>>() {
                     @ProcessElement
-                    public void processElement(ProcessContext c) {
-                      String word = c.element().getKey();
-                      Double df = c.element().getValue().getOnly(dfTag);
+                    public void processElement(
+                        @Element KV<String, CoGbkResult> element,
+                        OutputReceiver<KV<String, KV<URI, Double>>> o) {
```

Comment:
Starting from here `outputReceiver` changed to `o`. Any reason for that ?
I would prefer to have consistent definition for text search.

Reply:
@clairemcginty @RustedBones if you don't mind: I would change it to `o` everywhere. This is the naming I saw in Beam's code samples

Reply:
My mistake, they name it `out`:
https://beam.apache.org/documentation/programming-guide/
I will rename accordingly